home/import_content_pages.py

```diff
@@ -543,10 +555,12 @@ def add_whatsapp_to_page(self, page: ContentPage) -> None:
         page.is_whatsapp_template = self.is_whatsapp_template
         page.whatsapp_title = self.whatsapp_title
         page.whatsapp_template_name = self.whatsapp_template_name
-        page.whatsapp_template_category = self.whatsapp_template_category
         page.whatsapp_body.clear()
         for message in self.formatted_whatsapp_body:
-            page.whatsapp_body.append(("Whatsapp_Message", message))
+            if isinstance(message, WhatsAppTemplate):
+                page.whatsapp_body.insert(0, ("Whatsapp_Template", message))
```

Comment:
We should be able to have linked templates be in any position in a content page, they don't have to be the first message in a content page anymore